### PR TITLE
Testing/14th and 15th may

### PR DESCRIPTION
**Pull request by murgui**
Be nice

### CODE REVIEW

File: Software/Development/SemesterProject/App/Include/PIController.h [hunk 1]
```diff
@@ -31,8 +32,7 @@ typedef struct PIobjectTag
  */
 void PIObject_Constructor(PIobject *PIcontroller, float KP, float KI, int antiWindupFlag, int saturationBlock);
 
-float PiCalculation(PIobject *PIcontroller, float reference, float measurementValue);
-
+float PiCalculation(PIobject *PIcontroller, float reference, float measuredValue, int PIIsIqController);
```
Comment (murgui) [inline Software/Development/SemesterProject/App/Include/PIController.h]: the name of the last variable changed to something like controllerToBePrinted

**Review by NicolaiFransen — APPROVED** [review]

File: Software/Development/SemesterProject/App/SVM_DMC.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+}
+
+void runSVM_DMC(alphaBetaObject abVoltages)
+{
+
+    svgen_dq.Ualpha = abVoltages.alphaComponent;     // Pass inputs to svgen_dq1
+    svgen_dq.Ubeta = abVoltages.betaComponent;       // Pass inputs to svgen_dq1
+
+    SVGEN_MACRO (svgen_dq);        // Call compute macro for svgen_dq1
```
Comment (murgui) [inline Software/Development/SemesterProject/App/SVM_DMC.c:36]: should this be a pointer?

File: Software/Development/SemesterProject/App/controller.c [hunk 3]
```diff
@@ -32,17 +32,17 @@ void initPIControllers(void)
  */
 float PiCalculationIQ(float reference, float measuredValue)
 {
-    return PiCalculation(&PIControllerList.IqController, reference, measuredValue);
+    return PiCalculation(&PIControllerList.IqController, reference, measuredValue, 0);
```
Comment (murgui) [inline Software/Development/SemesterProject/App/controller.c]: substitute by define

**Review by murgui — COMMENTED** [review]
A couple of things to be improved

File: Software/Development/SemesterProject/App/SVM_DMC.c (hunk 2 above)
Comment (murgui) [reply Software/Development/SemesterProject/App/SVM_DMC.c:36]: No, as it is a macro and the code is just substituted.

**Pull request merged**